cmd/appgw-ingress/main.go

```diff
@@ -161,14 +162,20 @@ func main() {
 
 	if err = azure.WaitForAzureAuth(azClient, maxAuthRetryCount, retryPause); err != nil {
```

Comment:
Can we please change the name  to some thing else. WaitForAppGateway ? CheckAppGwIsDeployed